lib/openfeature/rollout/provider.rb

```diff
@@ -51,8 +51,8 @@ def initialize(rollout_client: nil)
         @rollout_client = rollout_client
       end
 
-      def fetch_boolean_value(flag_key:, default_value:, evaluation_context: nil)
-        res = @rollout_client.active?(flag_key, evaluation_context&.targeting_key)
+      def fetch_boolean_value(flag_key:, default_value: nil, evaluation_context: nil)
```

Comment:
I can see nothing in the spec about this, so I think it's OK to add.  However, instead of it being `nil`, should we default to an actual boolean value, either `true` or `false` to avoid getting into any strange tri-value boolean situations?

Reply:
Probably a good idea. I've gone with `false` in [b838b96](https://github.com/bryanalves/openfeature-rollout-provider/pull/1/commits/b838b965b9201db728e58c7a474411fd894cd1d2), but let me know if you're aware of a good reason to go with `true`